From ab9bcecb989aa7e5663e1bb3b32bfe1bc1ff1f09 Mon Sep 17 00:00:00 2001
From: Kentaro Hayashi <hayashi@clear-code.com>
Date: Tue, 25 Jul 2017 18:13:41 +0900
Subject: [PATCH] debsign: simplify signed report

In the previous versions, if there are huge number of signed files,
debsign prints "Successfully signed dsc, dsc, ... files" mechanically.

It is too redundant output. it is good enough to print total signed number.

Before: debsign against 100 files

  Successfully signed dsc, dsc, dsc, ...(print dsc up to 100 times) files

After: debsign against 100 files

  Successfully signed 100 files

Signed-off-by: Kentaro Hayashi <hayashi@clear-code.com>
---
 scripts/debsign.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/debsign.sh b/scripts/debsign.sh
index 924b03eb..9d1c2ad9 100755
--- a/scripts/debsign.sh
+++ b/scripts/debsign.sh
@@ -586,9 +586,9 @@ maybesign_changes() {
 
 report_signed() {
     if [ $NUM_SIGNED -eq 1 ]; then
-	echo "Successfully signed $HAVE_SIGNED file"
+	echo "Successfully signed $NUM_SIGNED file"
     elif [ $NUM_SIGNED -gt 0 ]; then
-	echo "Successfully signed $HAVE_SIGNED files"
+	echo "Successfully signed $NUM_SIGNED files"
     fi
 }
 
-- 
2.13.3

